### DISCUSSION TITLE
General Question about setCoords()

### DISCUSSION THREAD

**Discussion by kartiksolanki** [fabricjs/fabric.js#7467] [Q&A]
After performing scaling or rotate transformation on GROUP, do i need to call setCoords() on just group or do I have to loop through all the child objects and call setCoords() on each of them?

**Comment by asturur** [comment]
How do you perform scale and rotate? if with mouse nothings needs to becalled.
If you call it by code, then yes call setCoords on the group.

The items inside the group have coordinates that are relative to the group and never change.

What are you doing with the inner oCoords?
